scripts/lxc-to-lxd

```diff
@@ -9,6 +9,80 @@ import subprocess
 import sys
 
 
+supported_keys = [
+        'lxc.arch',
+        'lxc.pts',
+        # 'lxc.tty',
+        # 'lxc.devttydir',
+        # 'lxc.kmsg',
+        'lxc.aa_profile',
+        'lxc.aa_allow_incomplete',
+        # 'lxc.se_context',
+        'lxc.cgroup',
+        # 'lxc.id_map',
+        'lxc.loglevel',
+        # 'lxc.logfile',
+        'lxc.mount.entry',
+        'lxc.mount.auto',
+        'lxc.mount',
+        # 'lxc.rootfs.mount',
+        # 'lxc.rootfs.options',
+        # 'lxc.rootfs.backend',
+        'lxc.rootfs',
+        # 'lxc.pivotdir',
+        'lxc.utsname',
+        # 'lxc.hook.pre-start',
+        # 'lxc.hook.pre-mount',
+        # 'lxc.hook.mount',
+        # 'lxc.hook.autodev',
+        # 'lxc.hook.start',
+        # 'lxc.hook.stop',
+        # 'lxc.hook.post-stop',
+        # 'lxc.hook.clone',
+        # 'lxc.hook.destroy',
+        # 'lxc.hook',
+        'lxc.network.type',
+        'lxc.network.flags',
+        'lxc.network.link',
+        'lxc.network.name',
+        'lxc.network.macvlan.mode',
+        'lxc.network.veth.pair',
+        # 'lxc.network.script.up',
+        # 'lxc.network.script.down',
+        'lxc.network.hwaddr',
+        'lxc.network.mtu',
+        # 'lxc.network.vlan.id',
+        # 'lxc.network.ipv4.gateway',
+        # 'lxc.network.ipv4',
+        # 'lxc.network.ipv6.gateway',
+        # 'lxc.network.ipv6',
+        'lxc.network.',
+        'lxc.network',
+        'lxc.cap.drop',
+        'lxc.cap.keep',
+        # 'lxc.console.logfile',
+        # 'lxc.console',
+        # 'lxc.seccomp',
+        'lxc.include',
+        'lxc.autodev',
+        'lxc.haltsignal',
+        'lxc.rebootsignal',
+        'lxc.stopsignal',
+        'lxc.start.auto',
+        'lxc.start.delay',
+        'lxc.start.order',
```

Comment:
Are you sure? The script further down makes use of the values set for those keys. It initializes:

```
boot.autostart
boot.autostart.delay
boot.autostart.priority
```